From f41c3ebbc15b15bc620e99df352de30c699b0d95 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Wed, 4 Aug 2010 11:16:56 +1000
Patch-mainline: 2.6.36
References: fate#310916
Git-commit: dc77de12dde95c8da39e4c417eb70c7d445cf84b
Subject: [PATCH 1047/2588] drm/radeon: tone down overchatty acpi debug
 messages.

On non laptop systems we'll see these the whole time, so make them
less important.

Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit dc77de12dde95c8da39e4c417eb70c7d445cf84b)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/radeon/radeon_acpi.c |    2 +-
 drivers/gpu/drm/radeon/radeon_kms.c  |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/radeon/radeon_acpi.c b/drivers/gpu/drm/radeon/radeon_acpi.c
index e366434..3f6636b 100644
--- a/drivers/gpu/drm/radeon/radeon_acpi.c
+++ b/drivers/gpu/drm/radeon/radeon_acpi.c
@@ -35,7 +35,7 @@ static int radeon_atif_call(acpi_handle handle)
 
 	/* Fail only if calling the method fails and ATIF is supported */
 	if (ACPI_FAILURE(status) && status != AE_NOT_FOUND) {
-		printk(KERN_INFO "failed to evaluate ATIF got %s\n", acpi_format_exception(status));
+		printk(KERN_DEBUG "failed to evaluate ATIF got %s\n", acpi_format_exception(status));
 		kfree(buffer.pointer);
 		return 1;
 	}
diff --git a/drivers/gpu/drm/radeon/radeon_kms.c b/drivers/gpu/drm/radeon/radeon_kms.c
index e5b7054..ddcd3b1 100644
--- a/drivers/gpu/drm/radeon/radeon_kms.c
+++ b/drivers/gpu/drm/radeon/radeon_kms.c
@@ -81,7 +81,7 @@ int radeon_driver_load_kms(struct drm_device *dev, unsigned long flags)
 	/* Call ACPI methods */
 	acpi_status = radeon_acpi_init(rdev);
 	if (acpi_status)
-		dev_err(&dev->pdev->dev, "Error during ACPI methods call\n");
+		dev_dbg(&dev->pdev->dev, "Error during ACPI methods call\n");
 
 	/* Again modeset_init should fail only on fatal error
 	 * otherwise it should provide enough functionalities
-- 
1.7.6

